packages/dendron-cli/src/commands/notes.ts

```diff
@@ -18,6 +18,8 @@ type CommandCLIOpts = {
   query?: string;
   cmd: NoteCommands;
   output?: NoteCLIOutput;
+  newName?: string;
```

Comment:
instead of `newName` or `newVault`, would use the following:
- `destFname`
- `destVaultName`

Rationale behind the proposal:
- we already have a convention for using `dest` as the target in some other cli commands
- we prefer options to be more specific (eg. vault is ambiguous, vault name makes it clear what we are looking for)